src/web/mjs/connectors/MangaSY.mjs

```diff
@@ -11,4 +11,19 @@ export default class MangaSY extends WordPressMadara {
 
         this.queryTitleForURI = 'meta[property="og:title"]';
     }
-}
\ No newline at end of file
+
+    async _getPages(chapter) {
+        const url = new URL(chapter.id, this.url);
+        const request = new Request(url, this.requestOptions);
+        const script = `
+            new Promise((resolve, reject) => {
+                var imgdata = JSON.parse(CryptoJS.AES.decrypt(chapter_data, wpmangaprotectornonce, {
+                    format: CryptoJSAesJson
+                }).toString(CryptoJS.enc.Utf8));
+                resolve(imgdata);
```

Comment:
May perform `JSON.parse` inside the script itself, so the outer method can directly return the `FetchUI` invocation without awaiting the intermediate result.